plugins/proofpoint_tap/Dockerfile

```diff
@@ -17,10 +17,4 @@ RUN python setup.py build && python setup.py install
 # User to run plugin code. The two supported users are: root, nobody
 USER nobody
 
-# PLGN-701: POC how we can inject an env var into the container to specify a new date
```

Comment:
gonna approve but double check we don't need this comment anymore

Reply:
This has been replaced by backfill logic. Comment moved to wiki for posterity.